package.json

```diff
@@ -15,9 +15,10 @@
     "format": "prettier --write '**/*.{js,jsx,json,ts,tsx,yml,md,html,scss,css}'",
     "format:check": "prettier --check '**/*.{js,jsx,json,ts,tsx,yml,md,html,scss,css}'",
     "postinstall": "scripts/post-install.sh",
-    "cypress:open": "./node_modules/.bin/cypress open",
-    "cypress:watch": "./node_modules/.bin/cypress open --config watchForFileChanges=true",
-    "cypress:run": "./node_modules/.bin/cypress run"
+    "cypress:open": "cypress open",
+    "cypress:watch": "cypress open --config watchForFileChanges=true",
+    "cypress:run": "cypress run",
+    "cypress:run:firefox": "cypress run --browser firefox"
```

Comment:
@Ninamma This [article here](https://www.freecodecamp.org/news/npm-vs-npx-whats-the-difference/) is a good read 😄 

By default if you run `npm run ${script}`, npm will add the binaries within the `./node_modules/.bin` to the `$PATH` so you don't have to do `./node_modules/.bin` as they will be found by default.  `npx` is a handy tool if you want to run something within your local `./node_modules/.bin` without adding a script, see article!